index.html

```diff
@@ -1,99 +1,128 @@
+      <img class="snap2" src="assets/images/tonic2.svg" alt="Snapshot of portfolio">
+      <img id="second_tonic" src="assets/images/tonic.png" alt="Tonic Project name">
+      <p class="select">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <img class="tags" src="assets/images/Tags.png" alt="Tag buttons">
+      <button class="btn1" type="button">See project</button>
+    </div>
 
-    <div>
-      <img id="bg" src="assets/images/bg.png" alt="background image">
+    <div id="section4" class="section_2">
+      <img class="snap2" src="assets/images/stories2.svg" alt="Snapshot of portfolio">
+      <img class="stories" src="assets/images/stories.svg" alt="Tonic Project name">
+      <p class="select">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <img class="tags" src="assets/images/Tags.png" alt="Tag buttons">
+      <button class="btn1" type="button">See project</button>
     </div>
-   
+  </section>
 
-  </body>
-</html>
+  <section id="about-me" class="about_me">
+    <div id="greeter">
+      <h1>About <br>myself</h1>
+    </div>
+    <br> <br>
+    <p id="myself-text">I’m a software developer! I can help you build a product ,
+      feature or website Look through some of my work and experience!
+      If you like what you see and have a project you need coded,
+      don’t hestiate to contact me.
+    </p>
+    <h3 id="connects">LET’S CONNECT</h3>
+    <img id="sm_icons" src="assets/images/social-media-icons.svg" alt="social-media icons">
+    <button id="btn2" type="button" disabled>Get my resume</button>
+    <p id="lang">Languages</p>
+    <ul class="lang-list">
+      <li><a href="https://www.w3schools.com/js/default.asp" target="_blank" rel="noopener">JavaScript</a></li>
+      <li><a href="https://www.w3schools.com/html/default.asp" target="_blank" rel="noopener">HTML</a></li>
+      <li><a href="https://www.w3schools.com/css/default.asp" target="_blank" rel="noopener">CSS</a></li>
+      <li><a href="https://www.w3schools.com/css/css_rwd_frameworks.asp" target="_blank" rel="noopener">Frameworks</a>
+      </li>
+      <li><a href="https://targetjobs.co.uk/careers-advice/job-descriptions/software-developer-job-description"
+          target="_blank" rel="noopener">Skills</a></li>
+    </ul>
+  </section>
+
+  <footer class="footer">
+    <div class="foot-image">
+      <h2 class="contactme">
+        Contact me
+      </h2>
+      <p class="contact-info">
+        If you have an application you are interested in developing,
+        a feature that you need built or a project that needs coding.
+        I’d love to help with it
+      </p>
+      <form action="https://formspree.io/f/form" method="post" id="form" class="contact-form">
+        <ul class="form-item">
+          <li class="username"><input class="contact_input one_line" type="text" placeholder="username" name="username"
+              maxlength="30" required></li>
```

Comment:
- [ ] [ _Optional_ ] Kindly make this input placeholder consistent with there's of the form. Please ensure it is capitalized.